oaknational/Oak-Web-Application
# chore: add axe a11y tools in development environment

## Description

- adds @axe-core/react for a11y concern logging during development


## How to test

1. Run locally with env var `NEXT_PUBLIC_AXE_A11Y_LOGGING=on` in `.env`
2. Stick a glaring a11y issue somewhere
3. You should see axe log it in the console when you land on that page

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/192560818-1e790e3d-5140-43d1-9285-96314f50659f.png)


## Checklist
